scripts/publish_prod.sh

```diff
@@ -28,17 +28,40 @@ if [ -z "$AWS_SESSION_TOKEN" ]; then
     exit 1
 fi
 
+# Read the current version
+CURRENT_VERSION=$(node -pe "require('./package.json').version")
+
+# Read the desired version
+if [ -z "$1" ]; then
+    echo "Must specify a desired version number"
+    exit 1
+elif [[ ! $1 =~ [0-9]+\.[0-9]+\.[0-9]+ ]]; then
+    echo "Must use a semantic version, e.g., 3.1.4"
+    exit 1
+else
+    VERSION=$1
+fi
+
+# Confirm to proceed
+read -p "About to bump the version from ${CURRENT_VERSION} to ${VERSION}, and publish. Continue (y/n)?" CONT
+if [ "$CONT" != "y" ]; then
+    echo "Exiting"
+    exit 1
+fi
+
 yarn login
 
+echo "Updating layer versions"
 ./scripts/generate_layers_json.sh
-yarn test
 
-PACKAGE_VERSION=$(node -pe "require('./package.json').version")
+echo "Bumping the version number and committing the changes"
+yarn version --new-version "$VERSION"
 
 echo 'Publishing to Node'
+yarn test
 yarn build
-yarn publish --new-version "$PACKAGE_VERSION"
+yarn publish --new-version "$VERSION"
 
-echo 'Tagging Release'
-git tag "v$PACKAGE_VERSION"
-git push origin "refs/tags/v$PACKAGE_VERSION"
\ No newline at end of file
+echo 'Pushing updates to github'
+git push origin master
```

Comment:
We will need to disable branch protection on master for this to work

Reply:
I believe admins are allowed for direct (non-forced) pushes by default? I can try it out.